### ISSUE TITLE
live session - is it possible?

### ISSUE THREAD

**Issue by OrionRandD** [qTox/qTox#6165]
Does qtox allow me to share my screen? Say, to show me typing a Vim/Emacs live session to my students?

**Comment by Diadlo** [comment]
Yes. You can select full screen or it's part in Audio/Video settings as video device

**Comment by OrionRandD** [comment]
I've searched the internet for a video showing this feature in qtox. But, there is no such video showing this feature. Even in youtube. The problem is that I do not have another device - only one laptop for me to see it into action. All the videos in the internet showing qtox only comment on general features. 
Do you know anyone showing this feature in a video tutorial?
Many thx for replying :)

**Comment by Diadlo** [comment]
ToxID for EchoBot from the video: 76518406F6A9F2217E8DC487CC783C25CC16A15EB36FF32E335A235342C48A39218F515C39A6

[Video "tutorial"](https://github.com/qTox/qTox/files/4735726/qtox_screen_share.zip)

**Comment by OrionRandD** [comment]
@Diadlo Great! Thx a lot for showing me this feature. I think it should be included in the official Qtox channel/site for people to see it. Especially teachers who are trying to fly away from zoom, skype and proprietary vid-conf software :)

**Comment by skeletonium** [comment]
@Diadlo Just to make sure, do you mind if I add this tutorial to the wiki as per OrionRandD's suggestion?

**Comment by Diadlo** [comment]
@skeletonium I'm not quite good at writing. Feel free to create it yourself if you want. But if it's really needed, I can try :)

**Comment by skeletonium** [comment]
@Diadlo Oh, no, I only meant to ask your permission to upload the video. Like this: https://wiki.tox.chat/clients/qtox#screen_sharing
I'll add credits in the file description

**Comment by Diadlo** [comment]
@skeletonium Oh.. sorry, misread your message. Do whatever you want with video

**Comment by OrionRandD** [comment]
@Diadlo Great idea. Perhaps you could expand the video a little more to also show the other qTox features. Since you got the permission to do so.   ;)

**Comment by skeletonium** [comment]
@OrionRandD Seems you meant me instead, but I will probably make some additional screenshots in the future, yes.

**Comment by OrionRandD** [comment]
@skeletonium Oh. Yeah. Sorry. 
Anyway, that will be very good add for the project  ;)

**Comment by Nokia808** [comment]
@Diadlo 
@skeletonium 
@sudden6 
Hi all. I would like to suggest on you the following: this nice & very needed feature currently located within GUI in hidden place. For that my suggestion is to detach it from current place in GUI & re-locate it within "Video call" GUI because this feature unusually needed through out whole length of video call/conference. User usually need to share screen at intermittent way, for example, to show them picture from an atlas or a scientific book ...... So, implement new feature to add "Share screen" option within GUI of video call is much better than current state. Currently, user need to leave chat window & go into setting & change video then return ... this non-perfect & hidden ....
If you agree with me about this please inform me, so that I will open new feature request issue.

@OrionRandD 
Do you tried Jitsi ? It is very cure & secure video conference & open source free of charge application. Though it is centralized, but has new end to end encryption option. You can even use it from browser without install any thing. See the following:
https://jitsi.org/
& bellow is the official instance of developers team:
https://meet.jit.si/
There are many other instances & every one can build heir/his instance (server), like these:
https://meet.infomaniak.com/
&
https://calls.disroot.org/
But I found that inspite "infomaniak" is a great company, the official developer team instance is better than other options that I know !
Please be ware: e2ee is not working currently with Firefox. It is only available on Chromium/Chrome browsers & their derivatives. It is only available on desktop application but not in mobile applications. You can download desktop application from this link:
https://github.com/jitsi/jitsi-meet-electron/releases
Also, please notice that currently the maximum capacity of https://meet.jit.si instance is only 75 person per video room. They working hardly to increase capacity to more than 100 .....

**Comment by OrionRandD** [comment]
On Sun, Jun 07, 2020 at 12:38:55AM -0700, Nokia808 wrote:
>    [1]@Diadlo
>    [2]@skeletonium
>    [3]@sudden6
>    Hi all. I would like to suggest on you the following: this nice & very
>    needed feature currently located within GUI in hidden place. For that
>    my suggestion is to detach it from current place in GUI & re-locate it
>    within "Video call" GUI because this feature unusually needed through
>    out whole length of video call/conference. User usually need to share
>    screen at intermittent way, for example, to show them picture from an
>    atlas or a scientific book ...... So, implement new feature to add
>    "Share screen" option within GUI of video call is much better than
>    current state. Currently, user need to leave chat window & go into
>    setting & change video then return ... this non-perfect & hidden ....
>    If you agree with me about this please inform me, so that I will open
>    new feature request issue.

I totally agree with this new feature. I also find a little annoying of
having to leave chat window and go into settings to share the desktop. 
And I also had thought about this proposal with myself: "Why they do not
implement a easier way to share the desktop?". I hope that this
suggestion works

> 
>    [4]@OrionRandD
>    Do you tried Jitsi ? It is very cure & secure video conference & open
>    source free of charge application. Though it is centralized, but has
>    new end to end encryption option. You can even use it from browser
>    without install any thing. See the following:
>    [5]https://jitsi.org/
>    & bellow is the official instance of developers team:
>    [6]https://meet.jit.si/
>    There are many other instances & every one can build heir/his instance
>    (server), like these:
>    [7]https://meet.infomaniak.com/
>    &
>    [8]https://calls.disroot.org/
>    But I found that inspite "infomaniak" is a great company, the official
>    developer team instance is better than other options that I know !
>    Please be ware: e2ee is not working currently with Firefox. It is only
>    available on Chromium/Chrome browsers & their derivatives. It is only
>    available on desktop application but not in mobile applications. You
>    can download desktop application from this link:
>    [9]https://github.com/jitsi/jitsi-meet-electron/releases
>    Also, please notice that currently the maximum capacity of
>    [10]https://meet.jit.si instance is only 75 person per video room. They
>    working hardly to increase capacity to more than 100 .....

Yes. I know Jitsi very well, including the electron app for it. But, my
problem with Jitsi, for now, is my low bandwidth. That's why, for me,
ate the moment, qTox is a better option. It is simpler, more secure,
and, since I do not need to set a room for many people by now, it fits
me well. But, thx for trying to help me again :)

**Comment by skeletonium** [comment]
@Nokia808
Personally, I do not use video calls and desktop sharing myself, but it seems it would be a better UI for user experience. So if you want to make a feature request for it, I agree that you have the right idea.

**Comment by dpriskorn** [comment]
I would also like a more easily discoverable "share screen" button in the calling UI. Maybe a separate call button "call with screen sharing" that asks if you want to select a region?
